plugin/trino-kafka/pom.xml

```diff
@@ -339,6 +339,20 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Please add comments why this file is excluded.

Reply:
outdated. i've replaced checkstyle exclusion by java source generation from protobuf.